doc/logos.md

```diff
@@ -1,42 +0,0 @@
-# Logos
-
-For documentation on using logos, see https://shields.io/docs/logos
-
-## Contributing Logos
-
-Our preferred way to consume icons is via the SimpleIcons logo. As a first port of call, we encourage you to contribute logos to [the SimpleIcons project][simple-icons github]. Please review their [guidance](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) before contributing.
```

Comment:
I think we do still need to cover this specific point in the docs somewhere

Reply:
Added that to a small section in CONTRIBUTING.md.